packages/mui-base/src/FocusTrap/FocusTrap.test.tsx

```diff
@@ -5,22 +5,28 @@ import { act, createRenderer, screen } from 'test/utils';
 import FocusTrap from '@mui/base/FocusTrap';
 import Portal from '@mui/base/Portal';
 
+interface GenericProps {
+  [index: string]: any;
+}
+
 describe('<FocusTrap />', () => {
   const { clock, render } = createRenderer();
 
-  let initialFocus = null;
+  let initialFocus: HTMLElement | null = null;
 
   beforeEach(() => {
     initialFocus = document.createElement('button');
     initialFocus.tabIndex = 0;
     document.body.appendChild(initialFocus);
     act(() => {
-      initialFocus.focus();
+      if (initialFocus) {
+        initialFocus.focus();
+      }
```

Comment:
```suggestion
      initialFocus!.focus();
```

\+ other occurrences below